Automattic/vip-design-system
# Fix <OptionRow>, <Button>, <Notice> styles and allow variant to be passed

## Description

- This PR fixes <OptionRow> styles
- Adds a new prop called "variant" where you can edit which heading will be used
- Fixes the Button styles for links

## Checklist


## Steps to Test

1. Pull down PR.
2. `npm run dev`.
3. Open http://localhost:6006/?path=/story/optionrow--default
4. Should act the same as before
